packages/core/src/dispatcher.ts

```diff
@@ -147,10 +163,12 @@ export class Dispatcher {
     let dagResult = null
     for (let retries = IPFS_GET_RETRIES - 1; retries >= 0 && dagResult == null; retries--) {
       try {
-        dagResult = await this._ipfs.dag.get(asCid, {timeout: IPFS_GET_TIMEOUT, path})
+        dagResult = await this._ipfs.dag.get(asCid, { timeout: IPFS_GET_TIMEOUT, path })
       } catch (err) {
         if (err.code == 'ERR_TIMEOUT') {
-          console.warn(`Timeout error while loading CID ${cid.toString()} from IPFS. ${retries} retries remain`)
+          console.warn(
```

Comment:
Looks like some formatting changes snuck in here, not sure why.  @ukstv, did we ever run prettier against the entire codebase so that everything would be formatted the same way?

Reply:
Git blame tells me that 15 days ago your commit did not include proper formatting.

https://github.com/ceramicnetwork/js-ceramic/commit/99da05c94cfdbf8472c7f7cf14a3b68227766b4a#diff-e522280df0e1c4c8fe4f157393e23872535ee7bab6d33a2930c26b18f7411d19R150

Reply:
ah okay, that was before I set up my IDE to do it automatically. Thanks